.eslintrc

```diff
@@ -1,14 +1,11 @@
 {
-  "extends": [
-    "eslint:recommended",
-    "plugin:promise/recommended",
-    "prettier"
-  ],
-  "plugins": [
-    "promise"
-  ],
+  "extends": ["eslint:recommended", "plugin:promise/recommended", "prettier"],
+  "plugins": ["promise"],
   "parserOptions": {
-    "ecmaVersion": 2019
+    "ecmaVersion": 8
   },
-  "root": true
+  "root": true,
+  "rules": {
+    "promise/no-nesting": "off"
```

Comment:
What's the reason to disable this rule? I don't see it being violated in the bynder extension. (I might've missed it)

Reply:
`extensions/scripts/build.js` violates this